action.yml

```diff
@@ -107,6 +112,19 @@ runs:
         --name buildkitd_${GITHUB_RUN_ID} \
         -v /run/buildkit:/run/buildkit:rw \
         moby/buildkit:v0.12.1
+  - name: Set up scripts
+    shell: bash
+    run: |
+      cat <<EOF > "${GITHUB_WORKSPACE}/before.sh"
```

Comment:
Could you move this into `${GITHUB_WORKSPACE}/.kraftkit/before.sh` -- it may be better not to dirty the top-level working directory, in case the user has this script in their repo. (Same for `after.sh`).